gmso/utils/ff_utils.py

```diff
@@ -398,6 +402,56 @@ def parse_ff_connection_types(connectiontypes_el, child_tag="BondType"):
     return connectiontypes_dict
 
 
+def parse_ff_pairpotential_types(pairpotentialtypes_el):
```

Comment:
Can this function be removed? And `parse_ff_connection_types` be used and it should be renamed to `parse_ff_potentials`.

Reply:
Sorry could you explain a bit more what you mean here?

Reply:
This is not absolutely necessary but the function `parse_ff_connection_types` already does what is acheived by this function. However, an argument could be made that since `PairPotential`s are different from `AngleType`, `BondType`  etc... the usage of this function is justified.

Reply:
I see. I personally would vote for keeping these two functions separate as `PairPotential` and `connection_types`, although similar in form, have a crucial difference of one toggled to atomtypes and the other toggled to connections. So I think it makes more sense to deal with them separately while possible. This might also be helpful if in the future we want to send the XML syntax of these two onto slightly branching directions. But on the other hand, you are right that the codes of these two functions are basically identical.

Reply:
Makes sense